ElectronicObserver.KancolleApi.Types/ApiReqKousyou/RemodelSlotlistDetail/APIReqKousyouRemodelSlotlistDetailRequest.cs

```diff
@@ -3,10 +3,10 @@
 public class ApiReqKousyouRemodelSlotlistDetailRequest
 {
 	[JsonPropertyName("api_id")]
-	public string ApiId { get; set; } = "";
+	public int ApiId { get; set; }
```

Comment:
Are the type changes for the request needed?

Reply:
Not anymore, but it makes sense for it to be an int here

Reply:
Would need to adjust request parsing, though. Requests use form data, so you need to parse query parameters, which currently makes all values string.

Reply:
I'm gonna revert those changes